plugins/search-backend-node/src/IndexBuilder.ts

```diff
@@ -59,15 +59,15 @@ export class IndexBuilder {
    * given refresh interval.
    */
   addCollator({
```

Comment:
Just one thing that came to mind when reading this, did you ever thing of changing this to something like `addCollatorFactory`, as its more a collator factory that gets aded now than a collator? 🤔

Reply:
Interesting!  Upon reflection, I wonder if it was previously kind of confusing ("Why do I need to pass a `collator` property to the `addCollator` method call? That seems redundant.")  ...And now it's at least less redundant.  ("Oh, I want to add a collator, but I can't just provide the collator, I have to provide a factory.")

Does that make sense?